raster-foundry/raster-foundry
# Fix node histogram color schemes

## Overview

Correctly order histogram breakpoints
Fix divide by 0 error
Start all color schemes at 0

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo
![image](https://user-images.githubusercontent.com/4392704/31293529-7fde53e2-aaa5-11e7-8580-60cb32b4c1c1.png)

## Testing Instructions

 * Verify that the max clipping checkbox on the color schemes now works
* Verify that the moveable breakpoints are the min and max ones

Closes #2606
